src/main/resources/css/reporting.css

```diff
@@ -207,3 +215,7 @@ pre {
     margin-left: 10px;
     margin-right: 10px;
 }
+
+* {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2496596343/issues/source?bid=3059308&fileBranchId=3269761#l219)